docs/customize/styling-theme.md

```diff
@@ -7,20 +7,19 @@ The theme comprises the header, footer and main color(s). We are going to modify
 Open the `assets/less/site/globals/site.variables` file and edit it to have the following:
 
 ``` less
-@import "@less/invenio_app_rdm/variables";
-
 @brandColor: /* your brand color here */ ;
-
 @navbar_background_image: unset;
 @navbar_background_color: @brandColor;
 @footerLightColor: @brandColor;
 @footerDarkColor: /* a shade of your brandColor */;
 ```
 
-!!! info "Less configuration in the future"
+Important:
```

Comment:
Not sure this `Important` will be much visible, what about:
```suggestion
!!! info Important:
```
or another ```!!! <tag>``` instead?